crates/objc2-encode/src/encoding.rs

```diff
@@ -248,6 +251,39 @@ impl Encoding {
     pub fn equivalent_to_box(&self, other: &EncodingBox) -> bool {
         compare_encodings(self, other, NestingLevel::new(), false)
     }
+
+    /// Computes the theoretical size in bytes of the represented value type,
+    /// considering everything is packed, i.e. no alignment is computed.
+    pub fn size(&self) -> usize {
```

Comment:
> This can fail to compute, so it should return an `Option<usize>`

You mean instead of returning `0` as a default? Then yes, I'll probably do that. The problem is in the use site: should we make the encoding check fail in this case, always succeed thus ignoring the case, or have a more explicit `.unwrap_or_default()` somewhere in between the two?

Reply:
I'd prefer `.unwrap_or(0)` at the use, rather than returning some incorrect value.